backend/controllers/commentsController.js

```diff
@@ -23,12 +23,12 @@ exports.getComments = asyncHandler(async (req, res, next) => {
 // @route       POST /api/v1/comments/vehicle/:vehicleId
 // @access      Private
 exports.createComment = asyncHandler(async (req, res, next) => {
-  
+  console.log("TESTTEST");
```

Comment:
This console log can be removed

Reply:
Thank you for flagging this. It's been removed in this [commit](https://github.com/purplesmurf1998/Autotracks/pull/91/commits/3d6230aa52a906de840c32c937d900b2d8777f78)